### ISSUE TITLE
[Issue] BIG ERROR.

### ISSUE THREAD

**Issue by zFunado** [GC-spigot/Tokens#92]
Hello good, the plugin has a big error with its Tile Expansion, this same when one breaks the block only gives the mined block, it does not also give the inventory the blocks that it breaks by the radius (3x3) which with Autosell and / as activated it does not work and it is exactly the same as mining without the TILE enchantment, here I have some tests in which even without the tile enchantment you gain more than with the enchantment.
Basically, in other words, TILE mine 3x3 (9 blocks) but of these 9 it only takes me to inventory 1, which is the one I mine.

https://youtu.be/7PA8lZUh5Dw
https://youtu.be/h2lg5g7XjRw
https://youtu.be/jBsXaMP29Mw
https://youtu.be/hbwODZwn-7w
